Front-End/challenge2/script.js

```diff
@@ -5,3 +5,96 @@ TO-DO:
 - The calculator should be completely functional
 
 */
+
+//Numbers
+let firstNumber = "";
+let secondNumber = "";
+
+//Arithmetic operator
+let operator = "";
+let action = true;
+
+//Show value
+let showValue = document.querySelector("#display");
+
+//Buttons with numbers
+document.querySelector("#zero").addEventListener("click", setNumber);
+document.querySelector("#one").addEventListener("click", setNumber);
+document.querySelector("#two").addEventListener("click", setNumber);
+document.querySelector("#three").addEventListener("click", setNumber);
+document.querySelector("#four").addEventListener("click", setNumber);
+document.querySelector("#five").addEventListener("click", setNumber);
+document.querySelector("#six").addEventListener("click", setNumber);
+document.querySelector("#seven").addEventListener("click", setNumber);
+document.querySelector("#eight").addEventListener("click", setNumber);
+document.querySelector("#nine").addEventListener("click", setNumber);
```

Comment:
This code could be improved, there are different type of selectors (not only by id), this way you can avoid duplicating the same code 10 times, let's say for example that you have 100 or 1000 numbers, how would you get the value? try to think what would be a better approach to get the values